mlflow/pyfunc/utils/data_validation.py

```diff
@@ -130,6 +130,14 @@ def _get_func_info_if_type_hint_supported(func) -> Optional[FuncInfo]:
             )
         else:
             return FuncInfo(input_type_hint=type_hint, input_param_name=input_param_name)
+    else:
+        # TODO: add link to documentation
```

Comment:
q: do you plan to resolve this TODO in a separate PR?

Reply:
Yes in the docs PR